refact-agent/gui/src/hooks/useCapsForToolUse.ts

```diff
@@ -174,6 +189,17 @@ export function useCapsForToolUse() {
     modelsSupportingTools,
   ]);
 
+  useEffect(() => {
+    if (!supportMetadata) {
+      dispatch(
+        setInitialAgentUsage({
```

Comment:
There might be an easier way to do this, I'll test it and send a link soon

Reply:
Seems like a similar situation to when `setInitalAgentUsage` was being called to much?